src/testPane.ts

```diff
@@ -173,7 +173,7 @@ function addTestNodes(
 }
 
 const codedTestFunctionName = "coded_tests_driver";
-const codedTestDisplayName = "Coded Tests";
+const codedTestDisplayName = " Coded Tests";
```

Comment:
Is this leading space just so it goes to the top of the list?

Reply:
yes